public/assets/plugins/code_view.min-cf2bed25e4a2ed96e8cc04fc31771566311d1555aa5d8ec50a4af806b312196f.js

```diff
@@ -0,0 +1,8 @@
ror&&"undefined"!=typeof CodeMirror&&(k=CodeMirror.fromTextArea(j.get(0),b.opts.codeMirrorOptions)),b.undo.saveStep(),b.html.cleanWhiteTags(!0),b.core.hasFocus()&&(b.selection.save(),b.$el.find('.fr-marker[data-type="true"]:first').replaceWith('<span class="fr-tmp fr-sm">F</span>'),b.$el.find('.fr-marker[data-type="false"]:last').replaceWith('<span class="fr-tmp fr-em">F</span>'),b.$el.blur());var d=b.html.get(!1,!0);b.$el.find("span.fr-tmp").remove(),d=d.replace(/<span class="fr-tmp fr-sm">F<\/span>/,"FROALA-SM"),d=d.replace(/<span class="fr-tmp fr-em">F<\/span>/,"FROALA-EM"),b.codeBeautifier&&(d=b.codeBeautifier.run(d,{end_with_newline:!0,indent_inner_html:!0,extra_liners:["p","h1","h2","h3","h4","h5","h6","blockquote","pre","ul","ol","table"],brace_style:"expand",indent_char:"	",indent_size:1,wrap_line_length:0}));var e,f;if(k){e=d.indexOf("FROALA-SM"),f=d.indexOf("FROALA-EM"),e>f?e=f:f-=9,d=d.replace(/FROALA-SM/g,"").replace(/FROALA-EM/g,"");var g=d.substring(0,e).length-d.substring(0,e).replace(/\n/g,"").length,h=d.substring(0,f).length-d.substring(0,f).replace(/\n/g,"").length;e=d.substring(0,e).length-d.substring(0,d.substring(0,e).lastIndexOf("\n")+1).length,f=d.substring(0,f).length-d.substring(0,d.substring(0,f).lastIndexOf("\n")+1).length,k.setSize(null,Math.max(c,150)),k.setValue(d),k.focus(),k.setSelection({line:g,ch:e},{line:h,ch:f}),k.refresh(),k.clearHistory()}else e=d.indexOf("FROALA-SM"),f=d.indexOf("FROALA-EM")-9,j.css("height",c),j.val(d.replace(/FROALA-SM/g,"").replace(/FROALA-EM/g,"")),j.focus(),j.get(0).setSelectionRange(e,f);b.$tb.find(" > .fr-command").not(a).addClass("fr-disabled"),a.addClass("fr-active"),!b.helpers.isMobile()&&b.opts.toolbarInline&&b.toolbar.hide()}function g(){var a=b.$tb.find('.fr-command[data-cmd="html"]');if(c())b.$box.toggleClass("fr-code-view",!1),e(a);else{b.popups.hideAll();var d=b.$wp.outerHeight();b.$box.toggleClass("fr-code-view",!0),f(a,d)}}function h(){c()&&(g(b.$tb.find('button[data-cmd="html"]')),j.val("").removeData().remove()),l&&l.remove()}function i(){if(!b.$wp)return!1;j=a('<textarea class="fr-code" tabindex="-1">'),b.$wp.append(j),j.attr("dir",b.opts.direction);var e=function(){return!c()};b.opts.toolbarInline&&(l=a('<a data-cmd="html" title="Code View" class="fr-command fr-btn html-switch'+(b.helpers.isMobile()?"":" fr-desktop")+'" role="button" tabindex="-1"><i class="fa fa-code"></i></button>'),b.$box.append(l),b.events.bindClick(b.$box,"a.html-switch",function(){g(b.$tb.find('button[data-cmd="html"]'))})),b.events.on("buttons.refresh",e),b.events.on("copy",e,!0),b.events.on("cut",e,!0),b.events.on("paste",e,!0),b.events.on("destroy",h,!0),b.events.on("form.submit",function(){c()&&(b.html.set(d()),b.events.trigger("contentChanged",[],!0))},!0)}var j,k,l;return{_init:i,toggle:g,isActive:c,get:d}},a.FroalaEditor.RegisterCommand("html",{title:"Code View",undo:!1,focus:!1,forcedRefresh:!0,callback:function(){this.codeView.toggle()}}),a.FroalaEditor.DefineIcon("html",{NAME:"code"})});
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected an identifier and instead saw '=='.<br>Expected an operator and instead saw 'typeof'.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>Mixed double and single quotes.<br>Identifier 'end_with_newline' is not in camel case.<br>Identifier 'indent_inner_html' is not in camel case.<br>Identifier 'extra_liners' is not in camel case.<br>Identifier 'brace_style' is not in camel case.<br>Identifier 'indent_char' is not in camel case.<br>Identifier 'indent_size' is not in camel case.<br>Identifier 'wrap_line_length' is not in camel case.<br>'f' is already defined.<br>Expected '{' and instead saw 'e'.<br>Expected '{' and instead saw 'b'.<br>Expected '{' and instead saw 'return'.<br>Too many errors. (88% scanned).